### DISCUSSION TITLE
How do I suppress this TS error? I have turned off the strict mode in the tsconfig.

### DISCUSSION THREAD

**Discussion by jzs11** [vuejs/language-tools#406] [Q&A] [answered]
Hi,

I am facing an annoying problem by using the VueUse's useVModel. The project sill can compile, but the TS server keeps reporting this as a syntactical error. 

![image](https://user-images.githubusercontent.com/2770401/130371797-9475e261-01ec-467e-a588-e24664bba67b.png)

![image](https://user-images.githubusercontent.com/2770401/130371785-e59c1e94-eda8-492a-9422-858d572431eb.png)

Any ideas I could get around with it?

Thanks

**Comment by yaegassy** [comment]
@jzs11 You may be able to suppress it with `@ts-ignore`. Try it. 

See: <https://github.com/johnsoncodehk/volar/issues/342#issuecomment-891936630>

**Reply by jzs11** [reply]
Hey, thanks for your reply. Here is some screenshots, maybe I mis-use it.

![image](https://user-images.githubusercontent.com/2770401/130373978-33273e04-6279-4f31-b62a-fe774bd9f176.png)

![image](https://user-images.githubusercontent.com/2770401/130373987-f0c55e32-a1dd-4007-b6c7-e0cf22140c60.png)

**Reply by yaegassy** [reply]
How about this?

```vue
<PersonDropdown
  v-else-if="columnHelper.isPerson(cell)"
  :cell="cell"
  :data="cell.value"
  // @ts-ignore
  v-model:data="cell.value"
/>
```

**Reply by jzs11** [reply]
ESLint is not happy with it. sorry.
![image](https://user-images.githubusercontent.com/2770401/130374283-0444f3da-b8ab-44f2-8bac-1bbb713dd609.png)

Just a bit annoying to see those red lines floating around the IDE, the project is still working fine. 😉

**Comment by johnsoncodehk** [comment, marked as answer]
Hi @jzs11, `v-model:data="xxx"` is shorthand of `:data="xxx"` + `@update:data="xxx = $event"`, when you use `:data` + `v-model:data`, here actually has double `:data` binding. So you just need to remove `:data` only keep `v-model:data`.

**Reply by jzs11** [reply]
Hi, I tried that before, the wired thing is if I remove v-model:data it stops binding the update event back to the parent which mean it stops working.

**Reply by jzs11** [reply]
I also created an issue in VueUse repo to ask any suggestion - https://github.com/vueuse/vueuse/issues/688

**Reply by johnsoncodehk** [reply]
I mean remove :data not v-model:data. 😅

**Reply by jzs11** [reply]
You are the man!! Thanks!
![image](https://user-images.githubusercontent.com/2770401/130375141-6e711b05-8fb9-46e1-a0bc-5ebc5e380f5c.png)
